src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles.length > 0)
+    .map((styles) => styles.split(':'))
+    .reduce(
+      (object, styles) => ({
+        ...object,
+        [styles[0].trim()]: styles[1].trim(),
+      }),
+      {},
+    );
```

Comment:
The chained methods should have the same indentation to improve readability. Each method call should be on a new line with the same level of indentation.